pages/_app/index.jsx

```diff
@@ -28,11 +28,14 @@ class App extends NextApp {
     isAuthorized: false,
   }
 
+  store = initStore()
```

Comment:
It executes on the server, does not it? Should it be so?

Reply:
server has always [new store](https://github.com/oacore/dashboard/blob/master/store/index.jsx#L15). So it should be safe.